### PR TITLE
Add a small writing style guide

### PR DESCRIPTION
**Pull request by wilzbach**
As proposed on #1401, here's a proposal for a small writing guide to clarify common rules for contributors.

CC @AndrejMitrovic

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -99,3 +99,14 @@ make -f posix.mak apidocs-prerelease
...
 `$R/dlang.org/web/library-prerelease`.
+
+
+### Writing style guide
+
+* Use the _present_ tense (except for historic facts)
+* Use a _neutral_ tone (No strong narrator = no first-person ("I", "we") nor second-person ("you"))
+* Use a _formal_ style (No slang)
+* Use the [American English spelling](https://en.wikipedia.org/wiki/American_and_British_English_spelling_differences)
+* Use the [Oxford comma](https://en.wikipedia.org/wiki/Serial_comma) for coordinating conjunctions (C, C++__,__ and D)
```
Comment (AndrejMitrovic) [inline CONTRIBUTING.md]: Remind me again what are the double underscores about?

Comment (AndrejMitrovic) [inline CONTRIBUTING.md]: Ah you are emphasizing the comma. :) All good.

**Comment by AndrejMitrovic** [comment]
LGTM.

File: CONTRIBUTING.md (hunk 1 above)
Comment (wilzbach) [inline CONTRIBUTING.md]: > Ah you are emphasizing the comma. :) All good.

Yep ;-)
For the future:  GitHub also allows to directly preview Markdown. Just click on the "View" link (top right).

Comment (AndrejMitrovic) [inline CONTRIBUTING.md]: The reason it confused me is because I thought I was looking at a ddoc file. :smile:

**Comment by CyberShadow** [comment]
I feel like this has gotten in way too quickly.

1. I think this needs broader consensus.
2. As I argued on the similar Phobos PR, we actively need to avoid stuffing `CONTRIBUTING.md` with trivia. GitHub tells people to read this file before people submit changes, so this should only contain things that people **need** to read before submitting changes. just because someone somewhere one time submitted a PR that would've been avoided had he read the entirety of our guidelines does not make a case to burden all contributors with more guidelines that they must read.

I think this should be reverted. Feel free to put this on the wiki on a linkable page, though.

@andralex ?

**Comment by andralex** [comment]
Yah this doesn't quite add a lot of value yet. It would be useful to initiate it as a wiki page and merge it when it gets more significant.

**Comment by AndrejMitrovic** [comment]
Ok we could move it to the wiki. I'll revert it for now.

**Comment by CyberShadow** [comment]
> @CyberShadow: From that perspective (the link on PR/issue creation), the whole file should be scrapped, though, and replaced with a short bullet list of important things to consider (links to style, our processes, …).

Yep, that's just what I said here: https://github.com/dlang/phobos/pull/4585#issuecomment-239056002

**Pull request merged**